spec/poros/food_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+    it 'reads data into variables' do
```

Comment:
Nice PORO test! If this were a project it would probably make sense to put the data stub in a fixture since the ingredients line is so long, but for the purpose of this assessment it's totally fine